File: Csharp/DIAGONAL_NEGATIVOS.cs
internal class Program {
    private static void Main(string[] args) {
       
        int n, i, j, cont;
        
        Console.Write("Qual a ordem da matriz? ");
        n = int.Parse(Console.ReadLine());

        int[,] mat = new int[n, n];

        for(i=0; i<n; i++) {
            for(j=0; j<n; j++) {
                Console.Write("Elemento [" + i + "," + j + "]: ");
                mat[i,j] = int.Parse(Console.ReadLine());
            }  
        }
        Console.WriteLine("DIAGONAL PRINCIPAL: ");
        for (i = 0; i < n; i++) {
            Console.Write(mat[i,i] + " ");
        }
        Console.WriteLine();
        cont = 0;
        for (i = 0; i < n; i++) {
            for (j = 0; j < n; j++) {
                if (mat[i,j] < 0) {
                    cont++;
                }
            }

        }
           
     Console.WriteLine("QUANTIDADE NEGATIVOS = " + cont);

            
    }

}

File: PYTHON/DIAGONAL_NEGATIVOS.py
n = int(input("Qual a ordem da matriz? "))

mat: [[int]] = [[0 for i in range(n)] for j in range(n)]

for i in range(0, n):
    for j in range(0, n):
        mat[i][j] = int(input(f"Elemento [{i}, {j}]: "))

print("DIAGONAL PRINCIPAL: ")
for i in range(0, n):
    print(f"{mat[i][i]}  ", end="")
print()

cont = 0
for i in range(0, n):
    for j in range(0, n):
        if mat[i][j] < 0:
            cont = cont + 1

print("QUANTIDADE DE NEGATIVOS: ", cont)
